### ISSUE TITLE
[LOW] [Splits] [$500] Request – Unable to Request money twice to the same user you have no conversation with.

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#33487]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v1.4.16-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause-Internal Team
**Slack conversation:** @

Prerequisite: The new account you're requesting money from must have an updated user name in NewDot, ie. first name = Matt, last = test, as seen in Matt A's vid below.  
## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Log in
3. Click the FAB menu in LHN
4. Click on "Request money" Manual.
5. Enter an amount into the BNP and click Next
6. Enter an email of the user you have no conversation with and click on the “Request …”
7. Go back to the LHN and create one more request with the same user (repeat steps 3-6)

## Expected Result:
Two requests are in DM with the user you just sent a request to.

## Actual Result:
Error appears when create the second request.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/a7f9f83a-06d9-4058-920a-da5ec4caae97

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01497816ffe49c0bd0</li>
        <li>Upwork Job ID: 1738134853342339072</li>
        <li>Last Price Increase: 2024-04-20</li>
    </ul>
</details>

**Comment by handaniel214** [comment]
This error is still exist?

**Comment by abdulrahuman5196** [comment]
Seems the issue was created near weekend. Will work on closing out review sooner.

**Comment by abdulrahuman5196** [comment]
@mallenexpensify Can we retest? I don't see this issue now

**Comment by mallenexpensify** [comment]
I don't either, going to close.  Thanks @abdulrahuman5196 
<img width="830" alt="image" src="https://github.com/Expensify/App/assets/22508304/af2d7ba0-2862-4047-a176-f3dc73d2956d">

<img width="1038" alt="image" src="https://github.com/Expensify/App/assets/22508304/8b61a664-49cf-420d-9ae3-8f3d3b82e1ed">

**Comment by lanitochka17** [comment]
Issue is still reproducible on the latest build  1.4.28-0


https://github.com/Expensify/App/assets/78819774/d47b8d00-c70a-47d6-a8eb-6a6b31f04bcb

**Comment by mallenexpensify** [comment]
Ah... snap, I figured it out (and updated the OP)
> Prerequisite: The new account you're requesting money from must have an updated user name in NewDot, ie. first name = Matt, last = test, as seen in Matt A's vid below.

https://github.com/Expensify/App/assets/22508304/a7ab37d8-6fd6-4606-a7ed-6456a0b768f9

@abdulrahuman5196 please review @bernhardoj 's proposal above, thx

**Comment by abdulrahuman5196** [comment]
Let me check with the new updated OP

**Comment by mallenexpensify** [comment]
Please review the proposal above @abdulrahuman5196

**Comment by abdulrahuman5196** [comment]
Will close out review today.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by abdulrahuman5196** [comment]
Reviewing now

**Comment by abdulrahuman5196** [comment]
I don't see the same issue. But for me it shows infinite loader and there is some issue with the second request.


https://github.com/Expensify/App/assets/46707890/d80c1368-79c9-45ed-b5e5-4ee69c81670d


Anyways we only have one proposal from @bernhardoj https://github.com/Expensify/App/issues/33487#issuecomment-1867641550

@bernhardoj All the issues linked in the proposal has been closed. Could you kindly check and let me know if the proposal is still valid to solve this issue? Maybe the infinite loader issue I am facing as well?
If not could you update if possible?

**Comment by bernhardoj** [comment]
We have an open issue for the infinite loader [here](https://github.com/Expensify/App/issues/34577) and the root cause is explained [here](https://github.com/Expensify/App/issues/34577#issuecomment-1894092638). I think it should be fixed on the BE though to not set the report to null.

**Comment by mallenexpensify** [comment]
Should I put this on hold pending the below?
- https://github.com/Expensify/App/issues/34577

**Comment by bernhardoj** [comment]
Yes, I think it's better to hold for it.

**Comment by mallenexpensify** [comment]
Put on hold, updated title, removed `Help Wanted` label and bumped to weekly.

**Comment by mallenexpensify** [comment]
On hold

**Comment by mallenexpensify** [comment]
STill holding on
- https://github.com/Expensify/App/issues/34577

**Comment by mallenexpensify** [comment]
Still on hold 
- https://github.com/Expensify/App/issues/34577

**Comment by mallenexpensify** [comment]
Off hold, added `retest-weekly`

**Comment by mallenexpensify** [comment]
Waiting for retest, adding to #vip-split

**Comment by mvtglobally** [comment]
Issue is reproducible during KI retests. 

![1711987915994!123321](https://github.com/Expensify/App/assets/43995119/bb4dcd74-75c0-4762-af2c-5a5ac3ee88d7)

**Comment by mallenexpensify** [comment]
Add `Help Wanted` label.

**Comment by mallenexpensify** [comment]
Waiting for proposals.

**Comment by lokesh-singhania** [comment]
Not able to reproduce this on Android (Chrome)

1. Don't see `Request Money` option (is some setting required for it?)
2. Tried to reproduce using `Track Expense` instead - but no issue found (recording below)
  
https://github.com/Expensify/App/assets/13049969/01f384bc-1b5d-444c-bf64-74208aa9baa5

Is this issue mac only?

**Comment by mallenexpensify** [comment]
Gonna close this, wasn't able to repro AND it's 5 months old, nav is totally different now. 

https://github.com/Expensify/App/assets/22508304/1a10bd7c-3ce6-4012-a8b6-074e0ab888be
